<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1333</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>908</width>
        <height>888</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QOpenGLWidget" name="model_viewer">
         <property name="minimumSize">
          <size>
           <width>836</width>
           <height>868</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>401</width>
        <height>888</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>401</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="group_list">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>420</y>
         <width>381</width>
         <height>291</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>13</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QPushButton" name="import_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>720</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
      <widget class="QPushButton" name="export_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>770</y>
         <width>361</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
      <widget class="QListWidget" name="object_list">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>381</width>
         <height>291</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>13</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="object_list_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Objects</string>
       </property>
      </widget>
      <widget class="QLabel" name="group_list_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>281</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Bahnschrift</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Groups</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
